package-releases/base/kustomization.yaml

```diff
@@ -3,8 +3,8 @@ kind: Kustomization
 resources:
   - cronjob.yaml
   - imagestream.yaml
-generatorOptions:
-  disableNameSuffixHash: true
+  - route.yaml
+  - service.yaml
```

Comment:
Route and service are not required, we won't deploy package-release-job :) Remove also the files accordingly please

Reply:
Thanks, force-pushed